### PR TITLE
5236 fix for modified name translation in correction/alteration

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#5236

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/aliases.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def update_aliases(business: Business, aliases: Dict) -> Dict:
     if modified_aliases := aliases.get('modified'):
         for current_alias in business.aliases.all():
             for mod_alias in modified_aliases:
-                if current_alias.alias == mod_alias.get('oldValue'):
+                if current_alias.alias == str(mod_alias.get('oldValue')).upper():
```
Comment (kialj876) [inline queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/aliases.py:34]: Can you add the .upper to both sides of this comparison? Not %100 sure the data will always be consistent when we load corps from COLIN so we might have some cases where current_alias.alias is mixed case

**Review by kialj876 — COMMENTED** [review]
Looks good, just one comment. Also could you add in a test case for a mixed case current_alias.alias and mixed case mod_alias.get('oldValue')?

File: queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/aliases.py (hunk 1 above)
Comment (vysakh-menon-aot) [reply queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/aliases.py:34]: Sure. I did not add based on the implementation below for "new" aliases

**Comment by lekshmimallika-aot** [comment]
Another point discussed for this ticket during grooming was the correction output may not show the edited ones as it reuses the IA template and shows only the new entries as of now.But I see that this was not listed in the ticket. Please create a new ticket to fix the correction IA output / you can fix in this one. Up to you.

**Review by lekshmimallika-aot — APPROVED** [review]

**Review by kialj876 — APPROVED** [review]
Nice work!

**Pull request merged**